client/src/pages/DuelOverview.jsx

```diff
@@ -99,10 +100,10 @@ const DuelOverview = ({
 
   const { currentUserScore, opponentScore, opponent, inProgress, finishedDate, TTL } = data;
 
-  const remainingTimeBeforeRemove = (function () {
-    if (!finishedDate) return TTL;
-    const timeDiff = new Date().getTime() - finishedDate;
-    return TTL - Math.floor(timeDiff / (1000 * 60 * 60 * 24));
+  const removeTime = (function () {
+    const d = increaseDate({ days: Number(TTL) }, new Date(finishedDate));
+    const up = setDateToNextHour(d, 0, 1);
+    return up;
   })();
```

Comment:
Can we remove the complexity of a immediate fonction call?
```suggestion
    const d = increaseDate({ days: Number(TTL) }, new Date(finishedDate));
    const removeTime = setDateToNextHour(d, 0, 1);
```

I'd also like to rename `removeTime` to `remainingTime`

Reply:
That's not the remaining time, but the date of deletion
I chose IIFE to keep unnecessary calculation variables out of the global scope, but I can remove the variable up, it was useful for my debugging ^^

Reply:
Oh! I misunderstood here. So maybe `removingTime` would be better? `removeTime` looks like a function to me.
Ok pour IIFE (and thanks for reminding me the acronym !)

Reply:
Oh yeah, I understand :) I will rename it